__tests__/app.test.js

```diff
@@ -117,4 +117,51 @@ describe("app", () => {
         });
     });
   });
+
+  describe("GET /api/articles/:article_id/comments", () => {
+    test("200: Should respond with an array of comments for given article_id requested with the most recent first", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((result) => {
+          expect(result.body.comments).toHaveLength(11);
+          expect(Array.isArray(result.body.comments)).toBe(true);
+          expect(result.body.comments).toBeSortedBy("created_at", {
+            descending: true,
+          });
+          result.body.comments.forEach((comment) => {
+            expect(comment).toHaveProperty("comment_id");
+            expect(comment).toHaveProperty("body");
+            expect(comment).toHaveProperty("article_id");
```

Comment:
I would check the properties value - eg we know votes is a number 
we know article_id in this case is 1 :)